volatility3/framework/symbols/windows/extensions/__init__.py

```diff
@@ -611,11 +611,22 @@ def is_valid(self) -> bool:
 
                 ctime = self.get_create_time()
                 if not isinstance(ctime, datetime.datetime):
+                    # A process must have a creation time
                     return False
 
                 if not (1998 < ctime.year < 2030):
                     return False
 
+                etime = self.get_exit_time()
+                if isinstance(etime, datetime.datetime):
+                    if not (1998 < etime.year < 2030):
```

Comment:
Not sure about limiting the max year to 2030?  I fear in 6 years someone's going to use this and complain.  Is there a reason we capped it at > 2030 ?  Could we make it 3000 or something outlandishly far in the future?

Reply:
I agree.  I followed the same approach we used earlier with `creation_time` line 617. 
Anyway, I'm with you; perhaps we should use 2050? The idea is to filter out weird stuff, so I wouldn't go too far beyond that.

Reply:
Ok, 2050 is better than 2030 (26 years is better than 6 years), but we probably want to leave a comment somewhere in here pointing out that if people come hunting in this file for a bug, it might be that legitimate values over a specific time are weeded out as being invalid.  Or we could really push the boat out, find the time the program is running and add X years onto that?  That'd at least keep the code from failing ever far off into the future?

Reply:
with "the time the program is running" you mean the current time?

Reply:
Yeah, so someone with their system time set to 2040 get years up to 2050 as valid?

Reply:
I know it seems a long way off, but I guarantee you no one will think of it except now or when it become or a problem...  5;P  However, we can leave some problems to future us if we're not that fussed...  5:)

Reply:
hehe.. yeah.. I know.. I just wonder if it could be an issue with some users running it,let's say, on a VM with wrong time set or something edge case like that.. it's like we are adding a dependency ... but I agree that will be the best in terms of code maintenance.. I like that part

Reply:
Well, ten years in the future is pretty unusual, and anything less than that should be accepted (and most people testing things set their clocks back), so if we go for "current year + 10".  It shouldn't add a dependency and it'll be better than digging into why it won't work in ten years time...  5:)

Reply:
It sounds good. Done